### DISCUSSION TITLE
Can connect with Netezza On premise Datawarehouse by ODBC driver or not ?

### DISCUSSION THREAD

**Discussion by RamesChan** [open-metadata/OpenMetadata#16751] [Q&A]
can I custom connection to connect with my companay data warehouse (Netezza on-premise) by ODBC or JDBC driver ?

if yes, can you give me example code ?

**Comment by Prajwal214** [comment]
@RamesChan https://openmetadata.slack.com/archives/C02B6955S4S/p1719239537644199

You can create a custom connector, refer to the example below for CSV Connector 

https://github.com/open-metadata/openmetadata-demo/tree/main/custom-connector#creating-a-custom-connector

https://docs.open-metadata.org/v1.4.x/connectors/custom-connectors
